### PR TITLE
Catch message buffer exceptions

### PR DESCRIPTION
**Pull request by TimEvens**
Catch message buffer exceptions to prevent aborts on received malformed packets.

### CODE REVIEW

File: src/quicr_client.cpp [hunk 1]
```diff
@@ -98,7 +98,14 @@ class QuicRTransportDelegate : public ITransport::TransportDelegate
     }
 
     messages::MessageBuffer msg_buffer{ data.value() };
-    client.handle(std::move(msg_buffer));
+
+    try {
+      client.handle(std::move(msg_buffer));
+    } catch (const std::exception& /* ex */) {
```
Comment (GhostofCookie) [inline src/quicr_client.cpp:104]: Since we don't care about the exception this could just be `catch(...)`

File: src/quicr_server.cpp [hunk 2]
```diff
@@ -299,23 +299,29 @@ QuicRServer::TransportDelegate::on_recv_notify(
...
+            server.handle_publish(context_id, mStreamId, std::move(msg_buffer));
+            break;
+          case messages::MessageType::Unsubscribe:
+            server.handle_unsubscribe(
+              context_id, mStreamId, std::move(msg_buffer));
+            break;
+          default:
+            break;
+        }
+      } catch (const std::exception& /* ex */) {
```
Comment (GhostofCookie) [inline src/quicr_server.cpp:321]: Same as other comment.

**Review by GhostofCookie — APPROVED** [review]
Looks good! Comments are optional

File: src/quicr_client.cpp (hunk 1 above)
Comment (TimEvens) [reply src/quicr_client.cpp:104]: I was thinking that we might want to use it sooner than later. :)

**Pull request merged**